### PR TITLE
feat: add `NewCreateBatchAuction` CLI command

### LINKED ISSUE #91: feat: add CLI command for `BatchAuction`

### PR DESCRIPTION
**Pull request by jaybxyz**
## Description

closes: #91 

## Tasks

- [x] Add `NewCreateBatchAuction` CLI command
- [x] Update `NewPlaceBid` CLI command 
- [x] Update `How-To/cli/README.md`
- [x] Make enum `BidType` in proto consistent

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by jaybxyz** [comment]
@jeonghoyeo7 Can you review this when you have time? Due to the time bandwidth, I couldn't spend too much time on writing `How-To/cli/README.md`; so information about `BatchAuction` and how to place a bid with certain bid type are lacking at this point. Let's get back to it when we are done with implementation.

**Comment by jaybxyz** [comment]
I created [this issue](https://github.com/tendermint/fundraising/issues/99) to take care of it later.

File: x/fundraising/types/auction.go [hunk 1]
```diff
@@ -466,3 +468,12 @@ func (ba BaseAuction) GetMaxBidAmount(bidder string) sdk.Int {
 	}
 	return maxBidAmt
 }
+
+func (ba BaseAuction) SetMaxBidAmount(bidder string, maxBidAmt sdk.Int) error {
```
Comment (jeonghoyeo7) [inline x/fundraising/types/auction.go:472]: Does this also lead to add this method in (type AuctionI interface) in the 02_state.md? (including GetMaxBidAmount)

**Review by jeonghoyeo7 — COMMENTED** [review]
It seems there is nothing to seem wrong or to modify. I just left one comment for the spec change.

File: x/fundraising/types/auction.go (hunk 1 above)
Comment (jaybxyz) [reply x/fundraising/types/auction.go:472]: Yes, let me add those in the docs. Thanks!

**Pull request merged**